apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        if self.cleaned_data['your_name']:
+            raise forms.ValidationError(
```

Comment:
The point is if you show an error you're making it obvious that the honeypot field was tripped. But if we're assuming that bots are super dumb then that won't matter necessarily.  We should definitely instrument this so we know if it's catching anything at all.